src/cf-icons/usage.md

```diff
@@ -37,33 +37,22 @@ Theme variables for setting the color and sizes throughout the project. Overwrit
 ```
 @cf-icon-prefix:                cf-icon;
 @cf-icon-path:                  'fonts';
-@cf-icon-ie7-support:           true;
 @cf-icon-border-color:          #eee;
 ```
 
-#### IE7 support
-
-IE7 support is added by using dynamic properties (CSS expressions).
-
-To turn off IE7 support for performance reasons,
-simply set
-
-```
-@cf-icon-ie7-support: false;
-```
-
-
 ## The basics
 
 The cf-icon-prefix class applies all shared icon styles including the font family.
 By default, this class will be named `cf-icon` but it can be changed in the settings.
-All icons must use two classes, one for this base class and another to set the font character. For example:
+All icons must use three classes, one for this base class, one for the chosen pseudo-element, and one to set the font character. For example:
```

Comment:
Minor: `set the fond character` ➡️ `select the desired icon`. Setting the font character is an implementation detail they can discover by looking at the source, if they're really curious.